MAINTAINERS.md

```diff
@@ -18,14 +18,16 @@ This document explains who the maintainers are (see below), what they do in this
 
 ## Current Maintainers
 
-| Maintainer               | GitHub ID                                           | Affiliation |
-| ------------------------ | --------------------------------------------------- | ----------- |
-| Anirudha Jadhav          | [anirudha](https://github.com/anirudha)             |   Amazon    |
-| Joshua Li                | [joshuali925](https://github.com/joshuali925)       |   Amazon    |
-| Alex Meizer              | [alexmeizer](https://github.com/alexmeizer)         |  Bit Quill  |
-| Guian Gumpac             | [guiangumpac](https://github.com/guiangumpac)       |  Bit Quill  |
-| Raymond Lum              | [raymond-lum](https://github.com/raymond-lum)       |  Bit Quill  |
-| Yury Fridlyand           | [Yury-Fridlyand](https://github.com/Yury-Fridlyand) |  Bit Quill  |
+| Maintainer               | GitHub ID                                                          | Affiliation |
+| ------------------------ | ------------------------------------------------------------------ | ----------- |
+| Anirudha Jadhav          | [anirudha](https://github.com/anirudha)                            |   Amazon    |
+| Joshua Li                | [joshuali925](https://github.com/joshuali925)                      |   Amazon    |
+| Guian Gumpac             | [guiangumpac](https://github.com/guiangumpac)                      |  Bit Quill  |
+| Raymond Lum              | [raymond-lum](https://github.com/raymond-lum)                      |  Bit Quill  |
+| Yury Fridlyand           | [Yury-Fridlyand](https://github.com/Yury-Fridlyand)                |  Bit Quill  |
+| Max Ksyunz               | [MaxKsyunz](https://github.com/MaxKsyunz)                          |  Bit Quill  |
+| Forest Vey               | [forestmvey](https://github.com/forestmvey)                        |  Bit Quill  |
+| Mitchell Gale            | [MitchellGale-BitQuill](https://github.com/MitchellGale-BitQuill)  |  Bit Quill  |
```

Comment:
So we're merging maintainers? :) This can be cleaned up later.

Reply:
We are merging everything!